rpc/client/rpc_test.go

```diff
@@ -37,6 +42,7 @@ func getLocalClient() *client.Local {
 func GetClients() []client.Client {
 	return []client.Client{
 		getHTTPClient(),
+		getHTTPClientCustom(),
```

Comment:
we don't have to test every single RPC method just to ensure providing custom client is possible. A single test where we use NewHTTPWithClient and call some method will do!